src/components/screens/forms/form-reply-screen.tsx

```diff
@@ -0,0 +1,278 @@
+      setReply(updatedReply);
+      setFormValues(updatedValues as any);
+    } catch (e) {
+      errorContext.setError(strings.errorHandling.adminReplyScreen.saveReply, e);
+    }
+
+    setLoading(false);
+  };
+
+  /**
+   * Method for setting field value
+   *
+   * @param fieldName field name
+   * @param fieldValue field value
+   */
+  const setFieldValue = (fieldName: string, fieldValue: FieldValue) => {
+    if (formValues[fieldName] !== fieldValue) {
+      setFormValues({ ...formValues, [fieldName]: fieldValue });
+    }
+  };
+
+  /**
+   * Event handler for PDF export button click
+   */
+  const onExportPdfClick = async () => {
+    if (!metaform.id) {
+      return;
+    }
+
+    setLoading(true);
+
+    try {
+      const pdf = await repliesApi.replyExport({
+        metaformId: metaform.id!,
+        replyId: replyId!,
+        format: "PDF"
+      });
+  
+      MetaformUtils.downloadBlob(pdf, "reply.pdf");
+    } catch (e) {
+      errorContext.setError(strings.errorHandling.adminReplyScreen.exportPdf, e);
+    }
+
+    setLoading(false);
+  };
+
+  /**
+   * Renders the form
+   */
+  const renderForm = () => {
+    if (!metaform) {
+      return null;
+    }
+
+    return (
+      <Form
+        contexts={ ["MANAGEMENT"] }
+        metaform={ metaform }
+        getFieldValue={ getFieldValue }
+        setFieldValue={ setFieldValue }
+        onSubmit={ onSave }
+      />
+    );
+  };
+
+  /**
+   * View setup
+   */
+  const setup = async () => {
+    setLoading(true);
+
+    try {
+      const foundMetaform = await metaformsApi.findMetaform({
+        metaformSlug: formSlug!
+      });
+
+      const replyData = await repliesApi.findReply({
+        metaformId: foundMetaform.id!,
+        replyId: replyId!
+      });
+
+      const processedReplyData = await MetaformUtils.processReplyData(foundMetaform, replyData, attachmentsApi) as any;
+      const preparedFormValues = MetaformUtils.prepareFormValues(foundMetaform, processedReplyData, keycloak);
+
+      if (preparedFormValues && foundMetaform && replyData) {
+        setFormValues(preparedFormValues);
+        setMetaform(foundMetaform);
+        setReply(replyData);
+      }
+    } catch (e) {
+      errorContext.setError(strings.errorHandling.adminReplyScreen.fetchReply, e);
+    }
+
+    setLoading(false);
+  };
+
+  /**
+   * Handle selected reply status change
+   * 
+   * @param event event
+   */
+  const handleReplyStatusChange = (event: React.ChangeEvent<{ value: string }>) => {
+    if (!reply || !event) {
+      return;
+    }
+
+    const updatedReplyData = { ...reply.data, status: event.target.value };
+    const updatedReply = { ...reply, data: updatedReplyData as any };
+    setReply(updatedReply);
+  };
+
+  const statusOptions = Object.keys(ReplyStatus).map(status => { return status; });
+  console.log(LocalizationUtils.getLocalizedStatusOfReply(statusOptions[0]));
```

Comment:
remove logs?

Reply:
Changing to comment TODO since the localization is not working yet